Jenkinsfile

```diff
@@ -800,10 +795,6 @@ class EsLocalBuildEnvironment extends BuildEnvironment {
 	String jdkTool
```

Comment:
Looks like this can be removed as well. Don't forget to remove it in the calls to `new EsLocalBuildEnvironment` as well.